e2e_tests/tests/command/test_tensorboard.py

```diff
@@ -159,3 +161,48 @@ def test_start_tensorboard_for_multi_experiment(tmp_path: Path, secrets: Dict[st
                 break
         else:
             raise AssertionError(f"Did not find {SERVICE_READY} in output")
+
+
+@pytest.mark.slow
+@pytest.mark.e2e_cpu
+def test_start_tensorboard_with_custom_image(tmp_path: Path) -> None:
+    """
+    Start a random experiment configured with the shared_fs backend, start a
+    TensorBoard instance pointed to the experiment with custom image, verify
+    the image has been set, and kill the TensorBoard instance.
+    """
+    with FileTree(tmp_path, {"config.yaml": shared_fs_config(1)}) as tree:
+        config_path = tree.joinpath("config.yaml")
+        experiment_id = exp.run_basic_test(
+            str(config_path), conf.fixtures_path("no_op"), num_trials
+        )
+    command = [
+        "det",
+        "-m",
+        conf.make_master_url(),
+        "tensorboard",
+        "start",
+        str(experiment_id),
+        "--no-browser",
+        "--detach",
+        "--config",
+        "environment.image=alpine",
+    ]
+    env = os.environ.copy()
+    env["DET_DEBUG"] = "true"
+    res = subprocess.run(
+        command, universal_newlines=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE, env=env
+    )
+    assert res.returncode == 0, "\nstdout:\n{} \nstderr:\n{}".format(res.stdout, res.stderr)
```

Comment:
Instead of asserting the returncode, just put `check=True` in the `subprocess.run()` call.